From d244cd1d328b973abb6ac9037aac426b6b9ca3cb Mon Sep 17 00:00:00 2001
From: Ziyang Huang <hzyitc@outlook.com>
Date: Sun, 20 Oct 2024 20:28:28 +0800
Subject: [PATCH 1/1] wifi: ath11k: correct ATH11K_PCI_IRQ_DP_OFFSET

In 4ab4693f327a ("ath11k: use ATH11K_PCI_IRQ_DP_OFFSET for DP IRQ"), we
use ATH11K_PCI_IRQ_DP_OFFSET to index DP IRQ.

But the value of (ATH11K_PCI_IRQ_CE0_OFFSET + CE_COUNT_MAX) is 15, not 14.

According to the irq_name of PCI, this seem like a typo.

Fixes: 4ab4693f327a ("ath11k: use ATH11K_PCI_IRQ_DP_OFFSET for DP IRQ")
Signed-off-by: Ziyang Huang <hzyitc@outlook.com>
---
 drivers/net/wireless/ath/ath11k/pcic.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/wireless/ath/ath11k/pcic.h b/drivers/net/wireless/ath/ath11k/pcic.h
index ac012e8..b276872 100644
--- a/drivers/net/wireless/ath/ath11k/pcic.h
+++ b/drivers/net/wireless/ath/ath11k/pcic.h
@@ -10,7 +10,7 @@
 #include "core.h"
 
 #define ATH11K_PCI_IRQ_CE0_OFFSET	3
-#define ATH11K_PCI_IRQ_DP_OFFSET	14
+#define ATH11K_PCI_IRQ_DP_OFFSET	15
 
 #define ATH11K_PCI_CE_WAKE_IRQ	2
 
-- 
2.40.1

